account.go

```diff
@@ -310,7 +312,53 @@ func CreateAccountXLMTransaction(from SeedStr, to AddressStr, amount string,
 			build.Destination{AddressOrSeed: to.String()},
 			build.NativeAmount{Amount: amount},
 		),
-		build.MemoText{Value: "via keybase"},
+		build.MemoText{Value: defaultMemo},
+	)
+	if err != nil {
+		return res, err
+	}
+	return sign(from, tx)
+}
+
+// AccountMergeTransaction creates a signed transaction to merge the account `from` into `to`.
+func AccountMergeTransaction(from SeedStr, to AddressStr,
+	seqnoProvider build.SequenceProvider) (res SignResult, err error) {
+	tx, err := build.Transaction(
+		build.SourceAccount{AddressOrSeed: from.SecureNoLogString()},
+		network,
+		build.AutoSequence{SequenceProvider: seqnoProvider},
+		build.AccountMerge(
+			build.Destination{AddressOrSeed: to.String()},
+		),
+		build.MemoText{Value: defaultMemo},
+	)
+	if err != nil {
+		return res, err
+	}
+	return sign(from, tx)
+}
+
+// RelocateTransaction creates a signed transaction to merge the account `from` into `to`.
+// Works even if `to` is not funded but in that case requires 2 XLM temporary reserve.
```

Comment:
Does this work if `to` is funded?

Reply:
Nope, if `toIsFunded` is set incorrectly* the tx will fail. Because one the one hand you can't 'merge' into an unfunded account. And on the other you can't 'create' a funded account.

*Or rather does not align with the network state at consensus when the tx is submitted :D

Reply:
Ah, I didn't see the `toIsFunded` parameter.